playbooks/roles/ocp-pod-disruption-budget/README.md

```diff
@@ -0,0 +1,61 @@
+OCP Pod disruption budget
+=======================================
+
+The role *"roles/ocp-pod-disruption-budget"* is an object to define the max disruption that can be caused to a collection of pods.
+
+
+
+Requirements
+------------
+
+ - Running OCP 4.x cluster.
+
+
+Role Variables
+--------------
+
+| Variable                       | Required | Default                   |Comments                                                                                                                                                                                                                                                                          |
+|--------------------------------|----------|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| pdb_enable        		     | no       | false                     | Flag to be set to true to enable pod disruption budget |   
+| pdb_minAvailable        		 | no       | 3                         | Number of pods must always be available, even during a disruption |                      
```

Comment:
There's also a maxUnavailable parameter. From a validation perspective, it would be useful to test it.

Reply:
Added the parameter (maxUnavailable)